apps/settings/js/downloads/download_item.js

```diff
@@ -87,9 +87,10 @@ window.DownloadItem = (function DownloadItem() {
     // Update the state properly in the element
     domElement.dataset.state = state;
     // Update content
-    updateContent(domNodes, download);
+    return updateContent(domNodes, download).then(() => {
+      return domElement;
+    });
```

Comment:
Since the return type has been changed to a Promise object, we need to update comments on Line#8-9, so people won't get it wrong.